README.md

```diff
@@ -23,68 +23,77 @@ Never submit security-related bugs through a Github Issue or by email.
 
 ## Requirements
 
-* Node 6.x LTS
+* You need [Node](https://nodejs.org/) 6.x which is the current
+  [LTS](https://github.com/nodejs/LTS) (long term support) release.
+* To make life easier, also install [yarn](https://yarnpkg.com/en/).
 
 The easiest way to manage multiple node versions in development is to use
 [nvm](https://github.com/creationix/nvm).
 
 ## Get started
 
-* npm install
-* npm run dev
-
-## NPM scripts for development
-
-Generic scripts that don't need env vars. Use these for development:
-
-| Script                  | Description                                           |
-|-------------------------|-------------------------------------------------------|
-| npm run dev:admin       |  Starts the dev server (admin app)                    |
-| npm run dev:amo         |  Starts the dev server and proxy (amo)                |
-| npm run dev:amo:no-proxy|  Starts the dev server without proxy (amo)            |
-| npm run dev:disco       |  Starts the dev server (discovery pane)               |
-| npm run flow:check      |  Check for Flow errors and exit                       |
-| npm run flow:dev        |  Continuously check for Flow errors                   |
-| npm run eslint          |  Lints the JS                                         |
-| npm run stylelint       |  Lints the SCSS                                       |
-| npm run lint            |  Runs all the JS + SCSS linters                       |
-| npm run version-check   |  Checks you have the minimum node + npm versions      |
-| npm test                |  Runs the unittest, servertests + lint                |
-| npm run unittest        |  Runs just the unittests                              |
-| npm run unittest:dev    |  Runs the unittests and watches for changes           |
-| npm run unittest:server |  Starts a unittest server for use with `unittest:run` |
-| npm run unittest:run    |  Executes unittests (requires `unittest:server`)      |
-| npm run servertest      |  Runs the servertests                                 |
+* type `yarn` to install all dependencies
+* type `yarn dev:amo` to start a dev server
```

Comment:
That's an interesting idea. I also wondered about us considering making the addons-frontend repo a single app and binning the separation between the disco pane and amo once admin is gone. It would remove a number of complications though we would need to retain them as separate entry points from a webpack perspective. That's a discussion for another time though.